package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint && npm run unit && npm run cover && npm run check-coverage && npm run snyk",
+    "test": "npm run lint && npm run unit && npm run cover && npm run check-coverage",
```

Comment:
I think I agree with you @daniel-ac-martin - the more I think about it the more I think that snyk should be a dependency of the end application, not the library. Even then though, it would pick up the vulnerability of the sub-dependency at that level so not sure the best way of getting around it